docs/usage.rst

```diff
@@ -2,8 +2,353 @@
+Input files to feature creation may thus be DNA or protein multi-FASTA files, which may optionally
+be gzipped.
+
+Feature creation is computationally demanding. For this reason, direct input of individual
+FASTA files to phenotrex is only implemented for prediction.
+For model training, evaluation (and for batch prediction), tabular files must be created from
+input FASTA files representing the genotype of all input files, encoded as eggNOG cluster IDs. This
+allows reuse of the created features for all training and evaluation purposes.
+
+To create a tabular genotype file suitable for use in phenotrex training:
+
+.. code-block:: console
+
+    $ phenotrex compute-genotype \
+        --out T3SS.train_eval.genotype \
+        --n_threads 4 \
+        train_eval/genomes/*.fna.gz
+
+After some time, this will create a new tab-separated values (TSV) file ``T3SS.train_eval.genotype``
+in the directory, of the following shape:
+
+.. code-block::
+
+    #feature_type:eggNOG5-tax-2
+    GCA_003096415.1.fna.gz	COG0656	COG0661	COG3161	COG0358	...
+    GCF_000006765.1.fna.gz	COG0593	COG3255	COG1195	COG0187	...
+    GCF_000006905.1.fna.gz	COG2202	COG0169	COG0237	COG0847	...
+    ...
+
+Here, all lines starting with ``#`` denote a metadata field read by phenotrex - for example the
+type of features, in this case eggNOG clusters for the NCBI taxon ID 2 (Bacteria) from eggNOG version 5.
+In each following line, the first field denotes the identifier of the input genome (the file name),
+followed by all features found in the genome, each separated by tabs.
+
+.. note::
+
+    Feature creation by phenotrex scales reasonably well with the number of threads
+    supplied to the ``compute-genotype`` command (``--threads``). However, for large sets of genomes
+    and when compute cluster resources are available to the user, it may be more expedient to compute
+    subsets of genomes in parallel on different machines, and concatenate them afterwards.
+
+
+Training of Phenotrex Classifiers
+---------------------------------
+To train a phenotrex classifier, two tabular input files are required: The ``genotype`` file
+(created from FASTA files in the last section), containing representations of the input genomes; and
+the ``phenotype`` file, containing true phenotypes for each input genome on which to train and
+evaluate the model. For this tutorial, we provide a phenotype file containing information on Type 3
+secretion system (T3SS) presence in each of the input genomes.
+
+The tabular phenotype file required for training and model evaluation has the following shape:
+
+.. code-block:: console
+
+    Identifier	T3SS
+    GCF_000012905.2.fna.gz	NO
+    GCF_000195735.1.fna.gz	NO
+    GCF_000060345.1.fna.gz	NO
+    GCF_000220235.1.fna.gz	NO
+    GCF_000190695.1.fna.gz	NO
+    GCF_000015365.1.fna.gz	NO
+    GCF_000173115.1.fna.gz	NO
+    GCF_000173095.1.fna.gz	NO
+    GCA_003096415.1.fna.gz	NO
```

Comment:
Should we include at least one `YES` sample?

Reply:
I was wondering already, I simply took the head of the file but you may be right